### ISSUE TITLE
when git remote is http parser errors

### ISSUE THREAD

**Issue by snags88** [learn-co/learn-test#24]
The regex on this line is expecting https, but if the remote is set to http for whatever reason, it errors - https://github.com/learn-co/learn-test/blob/master/lib/learn_test/repo_parser.rb#L14

error:
```
/usr/local/opt/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/learn-test-2.0.0/lib/learn_test/repo_parser.rb:14:in `get_repo': undefined method `[]' for nil:NilClass (NoMethodError)
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/learn-test-2.0.0/bin/learn-test:55:in `<top (required)>'
        from /usr/local/opt/rbenv/versions/2.1.5/bin/learn-test:23:in `load'
        from /usr/local/opt/rbenv/versions/2.1.5/bin/learn-test:23:in `<main>'
        from /usr/local/opt/rbenv/versions/2.1.5/bin/ruby_executable_hooks:15:in `eval'
        from /usr/local/opt/rbenv/versions/2.1.5/bin/ruby_executable_hooks:15:in `<main>'```

**Comment by snags88** [comment]
the other option is the just catch the http and output a friendly error message saying "must use https fore remote".

**Comment by loganhasson** [comment]
fixed by 11238a38efc64414578f827f9ffcd8b107873953
